clojure-mode.el

```diff
@@ -1947,7 +1947,7 @@ Returns a list pair, e.g. (\"defn\" \"abc\") or (\"deftest\" \"some-test\")."
 \"Non-logical\" sexp are ^metadata and #reader.macros."
   (comment-normalize-vars)
   (comment-forward (point-max))
-  (looking-at-p "\\^\\|#[[:alpha:]]"))
+  (looking-at-p "\\^\\|#:?:?[[:alpha:]]"))
```

Comment:
What's this supposed to match?

Reply:
(it seems to me like 0-2 `:`, but I think this can be expressed better with a different qualifier). I also don't see tests with `::`.

Reply:
At least not in the first test.

Reply:
> What's this supposed to match?

`^metadata` or 
`#reader-macro` or 
`#:ns-unqualified-reader-macro` or 
`#::namespaced-qualified-reader-macro`

Yes, it can be expressed as `#:{0,2}`

The test case uses `#::ns`, but I did not write cases for each permutation.

Reply:
Btw, there's a small problem with the regexp now - something like `^::` will also match. Probably it should be tightened.

Reply:
And I'd really use `:\{0,2\}` instead of `:?:?`.